Makefile

```diff
@@ -52,7 +59,16 @@ docker:
 	@echo ">> building docker image"
 	@docker build -t "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
 
-$(PROMU) promu:
+$(GOPATH)/bin/promu promu:
 	@GOOS= GOARCH= $(GO) get -u github.com/prometheus/promu
 
-.PHONY: all style format build test vet tarball docker promu
+$(GOPATH)/bin/staticcheck:
```

Comment:
This is not in the standard makefile, please keep in sync with upstream

Reply:
[staticcheck](https://staticcheck.io/) is used in several other exporters.

Reply:
It's not in the promu or prometheus makefiles, I don't want exporters to get their makefiles out of sync.